documentation/api/v1/session_event_api_doc.rb

```diff
@@ -12,6 +12,10 @@ module SessionEventApiDoc
   LOCAL_PATH_EXAMPLE = '/path/to/file'
   REMOTE_PATH_DESC = 'Path to the associated file for upload, download, and filedelete events.'
   REMOTE_PATH_EXAMPLE = '/path/to/file'
+  ORDER_ENUM = [
```

Comment:
Maybe make this a global variable and put it in `RootApiDoc` since it's used in multiple places.

Reply:
Good suggestion, I'm making these changes for order, limit and offset.